bin/README.md

```diff
@@ -0,0 +1,14 @@
+# benchwrapper
+
+benchwrapper is a lightweight gRPC server that wraps the storage library for
+bencharmking purposes.
+
+## Running
+
+```
+cd nodejs-storage
+npm install
+npm run compile
```

Comment:
Don't need to `npm run compile` anymore :)

Reply:
Done